rules/test.bzl

```diff
@@ -45,6 +43,22 @@ _APPLE_BUNDLE_ATTRS = {
     ]
 }
 
+# See: https://bazel.build/reference/be/common-definitions#common-attributes
```

Comment:
so if bazel's newer version contains new common attribute, we need to update this list right?

Reply:
Ideally yeah, however most of these attrs are unused